### PR TITLE
Modify form

### PR DESCRIPTION
**Pull request by KojoSwiss**
- Put First name and Last name on the same line

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -147,6 +147,8 @@ GEM
     public_suffix (4.0.6)
     puma (4.3.6)
       nio4r (~> 2.0)
+    pundit (2.1.0)
```
Comment (ericblandet) [inline Gemfile.lock:150]: pundit still here
please delete

File: Gemfile.lock [hunk 2]
```diff
@@ -147,6 +147,8 @@ GEM
...
+      activesupport (>= 3.0.0)
```
Comment (ericblandet) [inline Gemfile.lock:151]: is it pundit?

File: Gemfile.lock [hunk 3]
```diff
@@ -272,6 +274,7 @@ DEPENDENCIES
   pry-byebug
   pry-rails
   puma (~> 4.1)
+  pundit
```
Comment (ericblandet) [inline Gemfile.lock:277]: please delete

File: app/assets/stylesheets/pages/_index.scss [hunk 4]
```diff
@@ -2,3 +2,4 @@
 @import "home";
 @import "show";
 @import "login";
+@import "signup";
```
Comment (ericblandet) [inline app/assets/stylesheets/pages/_index.scss:5]: is it pundit?
can you delete?

File: app/controllers/application_controller.rb [hunk 5]
```diff
@@ -1,12 +1,3 @@
 class ApplicationController < ActionController::Base
   before_action :authenticate_user!
-  before_action :configure_permitted_parameters, if: :devise_controller?
```
Comment (ericblandet) [inline app/controllers/application_controller.rb:3]: is it pundit? or devise?

File: app/controllers/workspaces_controller.rb [hunk 6]
```diff
@@ -24,6 +24,7 @@ def show
 
   def new
     @workspace = Workspace.new
+    authorize @workspace
```
Comment (ericblandet) [inline app/controllers/workspaces_controller.rb:27]: this is pundit

File: app/views/workspaces/show.html.erb [hunk 7]
```diff
@@ -18,8 +18,10 @@
             <li><i class="fas fa-users workspace-icon"></i><%= @workspace.capacity %> Max </li>
             <li><i class="fab fa-bitcoin workspace-icon mg-btm-v-sm"></i><%= @workspace.daily_rate %>$ / Day</li>
             <li class="u-center-text">
-              <%= link_to "Edit Workspace", edit_workspace_path(@workspace), class:"btn btn-home btn-center mg-right-md"  %>
-              <%= link_to "Delete", workspace_path, method: :delete, data: {confirm: 'Are you sure?'}, class:"btn btn-home btn-center"  %>
+              <% if current_user == @workspace.user %>
```
Comment (ericblandet) [inline app/views/workspaces/show.html.erb:23]: pundit => delete?

File: test/policies/restaurant_policy_test.rb [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+
+  def test_create
+  end
+
+  def test_update
+  end
+
+  def test_destroy
+  end
+end
```
Comment (Simon-Busch) [inline test/policies/restaurant_policy_test.rb:18]: what is that ?

File: test/policies/restaurant_policy_test.rb [hunk 9]
```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
```
Comment (ericblandet) [inline test/policies/restaurant_policy_test.rb:1]: delete

File: test/policies/workspace_policy_test.rb [hunk 10]
```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
```
Comment (ericblandet) [inline test/policies/workspace_policy_test.rb:1]: delete

**Review by ericblandet — CHANGES_REQUESTED** [review]

**Review by Simon-Busch — CHANGES_REQUESTED** [review]
Is the branch is not 100% functional, we can't merge it.

File: app/controllers/application_controller.rb (hunk 5 above)
Comment (KojoSwiss) [reply app/controllers/application_controller.rb:3]: this is devise

**Comment by ericblandet** [comment]
unwished branch

**Pull request closed without merging**